gravitee-apim-console-webui/src/components/gio-user-menu/gio-user-menu.component.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+
+.gio-user-menu {
+  &__alert {
```

Comment:
what is this __alert with an empty div `<div class="gio-user-menu__alert"></div>` ?

Reply:
<img width="181" alt="Capture d’écran 2022-11-23 à 08 43 51" src="https://user-images.githubusercontent.com/1457497/203494345-4a6553db-2e08-4c46-a0fb-dd830428dd60.png">

I replaced by a badge, it's the number of notifications